lib/components/atoms/Tooltip/Tooltip.js

```diff
@@ -0,0 +1,61 @@
+// @flow
+/**
+ *
+ * Tooltip
+ *
+ */
+import { Component } from 'react';
+import styled from 'styled-components';
+import classNames from 'classnames';
+
+import styles from './Tooltip.style';
+import type { Props, State } from './types';
+
+class Tooltip extends Component<Props, State> {
+  static defaultProps = {
+    message: '',
+    position: '',
+    className: '',
+    tabIndex: '',
+  };
+
+  state: State = {
+    displayTooltip: false,
+  };
+
+  hideTooltip = () => {
+    this.setState({ displayTooltip: false });
+  };
+
+  showTooltip = () => {
+    this.setState({ displayTooltip: true });
+  };
+
+  render() {
```

Comment:
You should use react portal so that tooltip doesn't get cropped because of overflow hidden on parent.